### DISCUSSION TITLE
Videos with transparent background

### DISCUSSION THREAD

**Discussion by David-Reinhardt** [ManimCommunity/manim#3826] [Q&A] [answered]
How can I produce animation videos with a transparent background without having to modify the video afterwards with FFmpeg?

**Comment by uwezi** [comment, marked as answer]
https://docs.manim.community/en/stable/faq/general.html#can-manim-render-a-video-with-transparent-background

**Comment by David-Reinhardt** [comment]
Super. Schon das zweite Mal genau die gesuchte Antwort.

Vielen Dank!
